### DISCUSSION TITLE
Relative Icon paths for non Windows Store Installs

### DISCUSSION THREAD

**Discussion by CEbbinghaus** [microsoft/terminal#13287] [Q&A]
Hello WT team. This application has got to be hands down my favourite tool to use on windows and I have built a lot of my tooling around it. I am running into an issue with non-windows store installs not loading icons.

For the Windows Store version installed under `AppData\local\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe` any icons placed inside the LocalState folder (alongside the settings) can be accessed by using the `ms-appdata:///local/iconname.png` shorthand. This is especially useful for me since I use a setup script that just system links the LocalState directory to one tracked by a repository. However, I have found that the Version of Windows Terminal installed by scoop uses a different configuration location: `AppData\local\Microsoft\Windows Terminal` and therefore the `ms-appdata:///local/iconname.png` shorthand breaks. 

Is there a way to configure it so that in both cases the icon file will load relative to the settings file like the case is with the windows store version? I would really like to swap to the scoop version since I am starting to use it for all of my applications and this is one of the headaches I feel should have a simple solution.

**Comment by CEbbinghaus** [comment]
Any update on this? I am tempted to make it into a Issue since it feels like a defect rather than a intentional design decision.

**Reply by zadjii-msft** [reply]
_Sorry - we're still trying our best to get a handle on wrangling Discussions too._ 

I suppose this could _theoretically_ be possible to fix - with a decent amount of effort. At the time of loading an icon or image, we'd have to initially check if we're running in an un-packaged context, and the path starts with `ms-appdata:`. If it does, just trim off that part of the string and treat it as relative to the "package root" where the Terminal is installed (unpackaged).

That's my initial thought. I dunno how feasible that would be. Unpackaged installs aren't _super_ well supported - our stance so far has been "just make sure unpackaged still runs". We don't have a lot of other engineering cycles to support more than that. 

If you wanna file a PR to try and fix that, I'm all ears.

**Reply by CEbbinghaus** [reply]
Sweet, I might look into adding this. Might take a bit as I am snowed under with work but I will pin this as a TODO for myself
